Plugins/ConnectMocksPlugin/ConnectMockGenerator.swift

```diff
@@ -219,4 +224,28 @@ private extension MethodDescriptor {
             """
         }
     }
+
+    func callbackAvailabilityAnnotation() -> String? {
+        if self.options.deprecated {
+            // swiftlint:disable line_length
+            return """
+            @available(iOS, introduced: 12, deprecated: 12, message: "This function has been marked deprecated.")
```

Comment:
Thoughts on changing these messages to "This RPC has been marked as deprecated in its `.proto` file."?